src/Stream.php

```diff
@@ -234,7 +236,7 @@ public function isReadable()
         $meta = stream_get_meta_data($this->resource);
         $mode = $meta['mode'];
 
-        return (strstr($mode, 'r') || strstr($mode, '+'));
+        return (strpos($mode, 'r') !== false || strpos($mode, '+') !== false);
```

Comment:
Same comment as above regarding `strstr` vs `strpos`. I recognize that `strpos` is marginally faster, but most folks agree it's a micro-optimization at best. If using it sacrifices readability, I'm not sure the change is useful.